examples/qwik/src/routes/index.tsx

```diff
@@ -0,0 +1,109 @@
+import { component$, useServerMount$, useSignal } from '@builder.io/qwik'
+import { grafbaseClient } from '~/utils/grafbase'
+
+interface Plants {
+  plantCollection: {
+    edges: Plant[]
+  }
+}
+
+interface Plant {
+  id: string
+  name: string
+  description: string
+}
+
+export const GetAllPlantsQuery = `
+  query GetAllPlants($first: Int!) {
+    plantCollection(first: $first) {
+      edges {
+        node {
+          id
+          name
+          description
+        }
+      }
+    }
+  }
+`
+
+export const AddNewPlantMutation = `
+  mutation AddNewPlant($name: String!, $description: String!) {
+    plantCreate(input: { name: $name, description: $description }) {
+      plant {
+        id
+        name
+        description
+      }
+    }
+  }
+`
+
+export default component$(() => {
+  const newPlant = useSignal('')
+  const newPlantDescription = useSignal('')
+  const allPlants = useSignal<Plants>()
+
+  useServerMount$(async () => {
+    const plants: Plants = await grafbaseClient({
+      query: GetAllPlantsQuery,
+      variables: { first: 100 }
+    })
+    allPlants.value = plants
```

Comment:
I don't know much about `.value` but we could satisfy the response type above in my example to we just return the edges as the value, making the types simpler for the signal:

```ts
messages.value = messageCollection?.edges || []
```